docs/DeveloperGuide.md

```diff
@@ -849,7 +852,7 @@ Step 3. CONNECTIONS will not display the first contact's invalid `Tag` and will
 
       Use case ends.
     
-* 1c. User inputs start index that is bigger than end index.
+* 1c. User inputs start index that is bigger then end index.
```

Comment:
wait what

Reply:
I think its than HAHAH cos its comparison not time

Reply:
Absolutely right... not sure what went on here.